rpc/block_delete.c

```diff
@@ -282,7 +282,7 @@ glusterBlockCleanUp(struct glfs *glfs, char *blockname,
   asyncret = glusterBlockDeleteRemoteAsync(blockname, info, glfs, &dobj, count,
                                            &drobj);
   if (asyncret) {
-    LOG("mgmt", GB_LOG_WARNING,
+    LOG("mgmt", GB_LOG_ERROR,
```

Comment:
I think these are in WARNING level coz in force delete this should not look bad may be ?

Hence lets leave them as it is ?

Thanks!

Reply:
Yeah, make sense. Will revert it back.